job-descriptions/engineering/engineering-manager-campaigns.md

```diff
@@ -0,0 +1,47 @@
+![logo](https://sourcegraph.com/.assets/img/sourcegraph-light-head-logo.svg)
+
+# Engineering Manager - Campaigns
+
+We are looking for an experienced engineering leader to manage our cross-functional engineering team building [campaigns](https://about.sourcegraph.com/handbook/engineering/campaigns). You will partner with our [product team](https://about.sourcegraph.com/handbook/product) to define team goals, and develop a roadmap to achieve those goals. Our product and team is very technical; we expect you to have a strong background as an engineer so that you can contribute to technical discussions and contribute code when necessary/appropriate. The team is small today but there is a lot more work to be done and we will expect you to grow the team in a sustainable way that maintains our high standards.
+
+## Qualifications
+
+- Minimum 3 years of experience as a software engineer plus 2 years of experience as an engineering manager.
+- Experience owning and achieving metrics driven objectives.
+- Experience working with one or more of languages that we use (e.g., Go, TypeScript/JavaScript).
+- Track record of recruiting, developing, and retaining top engineering talent.
+- Excellent communication skills, especially in writing.
+
+## Nice-to-haves
+
+- A network of excellent software engineers who would be interested to work with you again at Sourcegraph.
+- Published blog posts and/or tech talks about your work.
+
+## Learn more about us
+
+Sourcegraph is an equal opportunity workplace; we welcome people from all backgrounds and communities.
+
+To create a product that serves the needs of all developers, we are building a diverse [all-remote team](https://about.sourcegraph.com/company/remote) that is [distributed across the world](https://about.sourcegraph.com/company/team).
+
+We provide [competitive compensation](https://about.sourcegraph.com/handbook/people-ops/compensation) and [practical benefits](https://about.sourcegraph.com/handbook/people-ops/benefits-and-perks) to keep you happy and healthy so that you can do your best work.
+
+Learn more about what it is like to work at Sourcegraph by reading [our handbook](https://about.sourcegraph.com/handbook/).
+
+## Interview process
+
+1. You [apply here](TODO).
+1. You speak with our VP of Engineering for 45m.
+1. TODO: What project can we do? Coding? Technical writing?
```

Comment:
I'm wondering whether "coding" would make sense. Since they'd be working as a manager, I wouldn't expect a perfect submission (their coding skills might be rusty) and if we go in "expecting less", what do we expect?

Technical writing, on the other hand, could be insightful if we, for example, ask them to explain something really technical.